tsconfig.json

```
@@ -1,5 +1,10 @@
 {
   "compilerOptions": {
+    "baseUrl": "src",
+    "paths": {
+      "@pages/*": ["pages/*"],
```

let's do it this way:
```suggestion
      "@/*": ["src/*"],
```
it is more flexible because you do not need to add aliases for each folder.

---

also update `vite.config.ts`